frontend/styles/components/SimpleTable/simpleTableStyles.ts

```diff
@@ -0,0 +1,6 @@
+import { SxProps } from '@mui/material';
+
+/** Leaves enough space for the top header. */
+export const simpleTableStyles: SxProps = {
+  maxHeight: '80vh',
```

Comment:
@zognin is this the best way to do this?

Reply:
Seems so, the MUI documentation does this